packages/perseus/src/perseus-types.ts

```diff
@@ -789,7 +789,10 @@ export type PerseusGraphTypeCircle = {
     center?: Coord;
     radius?: number;
     // The initial coordinates the graph renders with.
-    startCoords?: Coord;
+    startCoords?: {
+        center: Coord;
+        radius: number;
+    };
```

Comment:
ADMITTEDLY, this is a bit janky. I tried to split them out into separate fields too, or use `radiusPoint` instead of radius, but those implementations got really tricky.

If anyone has any opinions or suggestions, I'd love to hear them!

Reply:
What's janky about it? It looks fine to me.

Reply:
Ah, I just didn't like that all the other `startCoord` types are arrays, but this one is an object. It also means the type is not as clean/streamlined when using it in other places - although I suppose I can make it its own type and import it if that were really an issue.